app/views/rooms/_form.html.erb

```diff
@@ -71,10 +71,6 @@
       </button>
     </div>
   </section>
-  <div class="form-group">
-    <%= f.label "Moderators" %>
-    <%= f.text_field :moderator_ids, class: "form-control room--moderators"  %>
-  </div>
```

Comment:
Can you keep this? One of my friends wants to add moderators while creating room 😄

Reply:
Sure. Can you explain what the expected input is here? Is it a comma-separated list of names?

Once I understand what is expected here, I can add a placeholder with an explanation and/or example input.

Reply:
It's an autocomplete input, see https://github.com/hlcfan/pokr/pull/101

Reply:
Hm, the placeholder is getting clipped for some reason, so I'm going to hold off on adding it for now

<img width="401" alt="create room pokrex 2017-10-07 21-37-14" src="https://user-images.githubusercontent.com/2379650/31313157-b311f2d4-aba7-11e7-957a-00bc27701f4e.png">